### DISCUSSION TITLE
Are there any plans to create android/iOS apps?

### DISCUSSION THREAD

**Discussion by joesteffee** [photoview/photoview#233] [Ideas]
This would be really great to replace something like google photos entirely. Not necessarily to upload photos but to view the whole library over the internet.

**Comment by viktorstrate** [comment]
Yes! I absolutely want to make mobile apps at some point.

It's not quite the same but the website can be installed as a Progressive Web App.
[How to install a PWA to your device](https://medium.com/progressivewebapps/how-to-install-a-pwa-to-your-device-68a8d37fadc1)

**Comment by gomme600** [comment]
Hi, any updates on this? It would be great to have an app like google photos that mixes local photos with the servers photos for a seemless galery.

**Reply by KatyPurry** [reply]
Good news, @viktorstrate released a iOS Beta app ! https://github.com/photoview/photoview/discussions/476

**Comment by baboonandsloth** [comment]
Hi,

not to be a pain, but any news on Android/Fdroid?

**Comment by viktorstrate** [comment]
Since the discussion started, an iOS app has been made that is available on the AppStore, and an Android app is in the makings.

https://github.com/photoview/photoview-ios
https://github.com/photoview/photoview-android
